sunpy/map/tests/test_mapbase.py

```diff
@@ -101,7 +108,9 @@ def simple_map():
     ref_pix = [1, 1] * u.pix
     scale = [2, 1] * u.arcsec / u.pix
     header = sunpy.map.make_fitswcs_header(data, ref_coord, reference_pixel=ref_pix, scale=scale)
-
+    header['crlt_obs'] = 0
```

Comment:
Instead of setting these explicitly, couldn't you just make this the observer of `ref_coord`?